HelloWorld/HelloWorld/Program.cs

```diff
@@ -10,8 +10,8 @@ internal class Program
 	{
 		private static void Main(string[] args)
 		{
-			Console.WriteLine("Hello World!");
```

Comment:
I've been to Australia and I've never heard anyone say G'day mate. Also, if we're going to be consistent with the previous greeting, "Mate" should be capitalized.